web_app/api/user.py

```diff
@@ -216,4 +217,22 @@ async def get_stats() -> GetStatsResponse:
     
     except Exception as e:
         logger.error(f"Error in get_stats: {e}")
-        raise HTTPException(status_code=500, detail=f"Internal server error: {str(e)}")
\ No newline at end of file
+        raise HTTPException(status_code=500, detail=f"Internal server error: {str(e)}")
+    
+@router.post("/allow-notification/{telegram_id}")
+async def allow_notification(
+    telegram_id: int,
+    telegram_db: TelegramUserDBConnector = Depends(lambda: TelegramUserDBConnector()),
+):
+    """Enable notifications for a specific telegram user"""
+    try:
+        user = telegram_db.allow_notification(telegram_id=telegram_id)
+        return {
+            "telegram_id": user.telegram_id,
+            "is_allowed_notification": user.is_allowed_notification,
+            "message": "Notifications enabled successfully"
```

Comment:
Please remove `telegram_id` and `is_allowed_notification` keys, we don't need them here